src/platform/src/common/features/download_insights_locations/components/Footer.jsx

```
@@ -0,0 +1,181 @@
+    hover: { scale: 1.03 },
+    tap: { scale: 0.98 },
+  };
+
+  return (
+    <div className="sticky bottom-0 left-0 right-0 bg-white border-t border-gray-200 px-4 py-3 sm:px-6 flex flex-col md:flex-row items-start md:items-center gap-2 justify-between shadow-lg z-10">
+      {/* Animated message area */}
+      <AnimatePresence mode="wait">
+        <motion.div
+          key={message}
+          className="text-sm leading-5 font-normal flex-1 mb-2 md:mb-0"
+          initial={{ opacity: 0, y: 5 }}
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)